### PR TITLE
feat: add CreateUtRequestContext function

### PR DESCRIPTION
**Pull request by li-jin-gou**
#### What type of PR is this?
feat

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
- 添加 CreateUtRequestContext 函数，用于辅助用户编写单测

### CODE REVIEW

**Review by songoow — CHANGES_REQUESTED** [review]
CreateUtRequestContext returns an app.RequestContext for testing purposes

File: pkg/common/ut/context.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import (
+	"io"
+	"io/ioutil"
+
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/protocol"
+)
+
+// CreateUtRequestContext returns a app.RequestContext for testing purposes
```
Comment (liu-song) [inline pkg/common/ut/context.go:27]: CreateUtRequestContext returns an app.RequestContext for testing purposes

Comment (li-jin-gou) [reply pkg/common/ut/context.go]: done

File: pkg/common/ut/context.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	"io/ioutil"
+
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/protocol"
+)
+
+// CreateUtRequestContext returns an app.RequestContext for testing purposes
+func CreateUtRequestContext(method, url string, body *Body, headers ...Header) *app.RequestContext {
+	ctx := app.NewContext(16)
+	var r *protocol.Request
```
Comment (welkeyever) [inline pkg/common/ut/context.go:30]: 30-42行逻辑和request.go相似度太高了，提炼一个函数出来公用吧

Comment (li-jin-gou) [reply pkg/common/ut/context.go:38]: done

**Review by Duslia — APPROVED** [review]

**Review by wzekin — APPROVED** [review]

**Pull request merged**